templates/ci/python-django.yml

```diff
@@ -0,0 +1,47 @@
+# Python Django
+# Test a Django app on multiple versions of Python.
+# https://aka.ms/yaml
+
+phases:
+
+# =================
+# Test a Django app
+# =================
+- phase: 'DjangoApp'
+  variables:
+    # relative path to your manage.py
+    projectPath: mysite/
+
+  queue:
+    name: 'Hosted Linux Preview'
+    parallel: 1
+    matrix:
+      Python35:
+        PYTHON_VERSION: '3.5'
+      Python36:
+        PYTHON_VERSION: '3.6'
+      Python37-dev:
+        PYTHON_VERSION: '>= 3.7.0a'
+  steps:
+
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(PYTHON_VERSION)'
+      architecture: 'x64'
+
+  - script: python -m pip install --upgrade pip setuptools wheel
+    displayName: 'Install dependencies'
+
+  - script: python -m pip install -r requirements.txt
+    displayName: 'Install requirements'
+
+  - script: python -m pip install unittest-xml-reporting
+    displayName: 'Install unittest-xml-reporting'
+
+  - script: pushd $(projectPath) && python manage.py test --testrunner xmlrunner.extra.djangotestrunner.XMLTestRunner --no-input
+    displayName: 'Run tests'
+  
+  - task: PublishTestResults@2
+    inputs:
+      testResultsFiles: "**/TEST-*.xml"
+      searchFolder: $(System.DefaultWorkingDirectory)/$(projectPath)
```

Comment:
Perhaps camel-case this like:
```
$(system.defaultWorkingDirectory)/$(projectPath)
```

Reply:
:+1: I borrowed this from another task. Will adapt to be more Unix-y.